packages/features/ee/workflows/pages/index.tsx

```diff
@@ -1,45 +1,72 @@
 import { useSession } from "next-auth/react";
+import { useRouter } from "next/router";
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.